test_module/test.py

```diff
@@ -0,0 +1,41 @@
+import test_module.test_coordinates as tc
+
+# Instructions
+# Start from basic_science -> ganesha_statue ------> basic_science
+
+
+class TestTree:
+    def __init__(self, gt, start_node, end_node):
+        self.G = gt
+        self.start_node = start_node
+        self.end_node = end_node
+
+        if self.start_node == 'basic_science' and self.end_node == 'ganesha_statue':
+            self.G.add_tree(tc.tree1.point_coordinate, start_node, end_node)
+            self.G.add_tree(tc.tree2.point_coordinate, start_node, end_node)
+            self.G.add_tree(tc.tree3.point_coordinate, start_node, end_node)
+            self.G.save_and_return()
+
+        elif self.start_node == 'ganesha_statue' and self.end_node == 'hostel_turn':
+            self.G.add_tree(tc.tree4.point_coordinate, start_node, end_node)
+            self.G.add_tree(tc.tree5.point_coordinate, start_node, end_node)
+            self.G.add_tree(tc.tree6.point_coordinate, start_node, end_node)
+            self.G.save_and_return()
+
+        elif self.start_node == 'hostel_turn' and self.end_node == 'volley_ball_court':
+            self.G.add_tree(tc.tree7.point_coordinate, start_node, end_node)
+            self.G.add_tree(tc.tree8.point_coordinate, start_node, end_node)
+            self.G.add_tree(tc.tree9.point_coordinate, start_node, end_node)
+            self.G.save_and_return()
+
+        elif self.start_node == 'volley_ball_court' and self.end_node == 'teacher_parking':
+            self.G.add_tree(tc.tree10.point_coordinate, start_node, end_node)
+            self.G.save_and_return()
+
+        elif self.start_node == 'teacher_parking' and self.end_node == 'basic_science':
+            self.G.add_tree(tc.tree11.point_coordinate, start_node, end_node)
+            self.G.add_tree(tc.tree12.point_coordinate, start_node, end_node)
+            self.G.save_and_return()
+
+        else:
+            pass
```

Comment:
Note that the conditions are based on the nodes in a loop. from basic_science and so on.

order to add trees on interface would be :
basic_science <-> ganesha_statue
ganesha_statue <-> hostel_turn
hostel_turn <-> volley_ball_court
volley_ball_court  <->  teacher_parking
teacher_parking  <->  teacher_parking